java/org/contikios/cooja/util/StringUtils.java

```diff
@@ -192,4 +193,15 @@ public static boolean saveToFile(File file, String text) {
       return false;
     }
   }
+
+  public static ArrayList<String> splitOnNewline(String commands) {
```

Comment:
What do you think about returning `List<String>` to avoid exposing implementation details?

Reply:
Fixed.